Shared/it.lproj/Localizable.strings

```diff
@@ -1015,50 +758,26 @@
 "Menu.Downloads.Label" = "Download";
 
 /* A switch to disable enhanced tracking protection inside the menu. */
-"Menu.EnhancedTrackingProtectionOff.Title" = "Protezione antitracciamento avanzata DISATTIVATA per questo sito.";
+"Menu.EnhancedTrackingProtectionOff.Title" = "Le protezioni sono DISATTIVATE per questo sito";
```

Comment:
Turns out, we never exposed the new string in localization 🤦🏼 (not just English). We really need to keep an eye out on export logs